tests/lib/prepare-restore.sh

```diff
@@ -531,6 +531,14 @@ prepare_project() {
         fi
         sleep 1
     done
+    # Update C dependencies
+    for _ in $(seq 10); do
```

Comment:
Why the for loop?

Reply:
I also wondered the same, and the answer I came up with is that this script also downloads the source code, so it might fail because of network issues.

Reply:
Hm but then it's missing `(.. || true)` isn't it? errexit is passed to subshells.

Reply:
We do the samd for "govendor sync" to protect against simple network issues (poor mans retry).

Reply:
This works because it's inside an `if` context or am I missing something here?

Reply:
Uhh yeah, sorry, I'm totally useless today.

Reply:
No worries :hug